Dockerfile

```diff
@@ -0,0 +1,46 @@
+FROM ubuntu:14.04
+MAINTAINER frikfry@gmail.com # Someone let me know an appropriate email for the project
+
+# Needed to fix pip install of requirements due to strange char encoding issue.
+ENV LC_CTYPE C.UTF-8
+RUN apt-get clean
+RUN apt-get update
+RUN apt-get install -y vim supervisor postgresql postgresql-contrib postgresql-client libpq-dev libffi-dev nodejs node npm python-pip python-dev python3-dev build-essential
```

Comment:
Please consider using node.js' recommended ppa (https://nodejs.org/en/download/package-manager/#debian-and-ubuntu-based-linux-distributions) as the default one may be quite out of date